code/modules/research/designs.dm

```diff
@@ -604,7 +604,45 @@
 	build_path = /obj/item/weapon/gun/energy/cutter/plasma
 	sort_string = "TAEAC"
 
-
+/datum/design/item/weapon/divet
+	name = "Winchester NK Divet pistol"
+	id = "divet"
+	req_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 5)
+	materials = list(MATERIAL_STEEL = 10000, MATERIAL_SILVER = 4000)
+	build_path = /obj/item/weapon/gun/projectile/divet
+	sort_string = "TACEA"
+
+/datum/design/item/weapon/divetslug
+	name = "divet magazine"
+	id = "divetslug"
+	req_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 4)
+	materials = list(MATERIAL_STEEL = 4000)
+	build_path = /obj/item/ammo_magazine/divet
+	sort_string = "TACEB"
+
+/datum/design/item/weapon/50cal
```

Comment:
50cal ammo? I don't believe we have any weapons which use this

Reply:
The magnum pistol does, I was originally going to make it cost deuterium, which is synthesized with oxyphoron (and is a rather flammable waste component, sadly, deuterium doesn't seem to have any material defines. It's basically a gambit to get more ammunition at a pretty high cost for the captain's pistol and it's pretty high combat tech to get, I thought it would be an interesting way to